homeassistant/components/keenetic_ndms2/config_flow.py

```diff
@@ -75,7 +76,9 @@ async def async_step_user(self, user_input=None):
             step_id="user",
             data_schema=vol.Schema(
                 {
-                    vol.Required(CONF_HOST): str,
+                    vol.Required(
+                        CONF_HOST, default=self.context.get(CONF_HOST, vol.UNDEFINED)
```

Comment:
```suggestion
                        CONF_HOST, default=self.context.get(CONF_HOST)
```

Ideally; we should not even ask for the data we got from discovery. So, I would hide the host field completely on a discovered item.

Reply:
Your suggestion will lead to 'None' being a default value in case of pure user setup. I'm passing vol.UNDEFINED here to say there is no default. Passing None leads to instant error message for missing host in UI even before user inputs anything

Reply:
As for not showing the field, I would still want the user to see the host which will be used

Reply:
The host that is used can be shown in e.g., the title of the discovered item, or even in the description of the flow.

Reply:
I think I got this working. Do I need to add `flow_title` for each transaction?